flowmachine/flowmachine/core/union.py

```diff
@@ -26,8 +26,8 @@ def __init__(self, *queries: Query, all: bool = True):
         column_set = set(tuple(query.column_names) for query in queries)
         if len(column_set) > 1:
             raise ValueError("Inconsistent columns.")
-        if len(queries) < 2:
-            raise ValueError("Union requires at least two queries.")
+        if len(queries) < 1:
+            raise ValueError("Union requires at least one query.")
```

Comment:
If we're allowing just one query, it's probably worth adding a warning if a user passes a single query with `all=False`, to warn that the output won't be de-duped (or, alternatively, handle this as a special case such that it _does_ de-dupe).

Reply:
Former I think - it would be weirdly different behaviour to dedupe a single query.